app/controllers/api/framework_responses_controller.rb

```diff
@@ -109,6 +109,8 @@ def render_bulk_update_error(exception)
     def send_notification
       if assessment.respond_to?(:amended_at) && assessment.amended_at.present?
         Notifier.prepare_notifications(topic: assessment, action_name: 'amend_person_escort_record')
```

Comment:
might we want to amend completed youth assessments in the future? if so, perhaps this line should be:
```
Notifier.prepare_notifications(topic: assessment, action_name: "amend_#{assessment.class.name.underscore}")
```
to avoid misnaming youth assessments as PERs ?

Reply:
Good idea, it's more future proof that way 👍